vars/standardJavaDeploy.groovy

```diff
@@ -66,6 +65,9 @@ def call(config) {
 
         stage('Build docker image') {
             container('docker') {
+                versionTag = "${appVersion}-${shortCommitSha}"
```

Comment:
If we have a version why are we still using the `shortCommitSha`?

Is this only to be able to tell from a glance at the version and not to enable the programmatic selection of the image version?

Reply:
We don't yet have version bump enforcement so this is to ensure that if someone forgets to bump the application version, we don't overwrite an existing image.